apps/laboratory/src/pages/library/wagmi.tsx

```diff
@@ -21,7 +21,8 @@ const modal = createAppKit({
   networks: ConstantsUtil.EvmNetworks,
   projectId: ConstantsUtil.ProjectId,
   features: {
-    analytics: true
+    analytics: true,
+    auth: true
```

Comment:
Why do we need this flag here if we already have the siwx param?
```suggestion
    auth: true
```